homeassistant/components/websocket_api/http.py

```diff
@@ -144,6 +147,7 @@ def handle_hass_stop(event):
 
             self._logger.debug("Received %s", msg)
             connection = await auth.async_handle(msg)
+            self._connections.add(connection)
```

Comment:
let's not add the connection, but instead store `id(self)`, which we also use for logging.

Reply:
That will work for this PR, although I was thinking of having a list of connected users in the dev-info page or somewhere, in which case keeping the connection set is needed.

Reply:
I think that it would be overkill really. Just because we can get all the data, doesn't mean we should 😆

Reply:
In which case it's just as well to keep a bare count, rather than a set of anything.

Reply:
How this even work? `WebSocketHandler` is not a singleton.

Reply:
Ahh then it won't work! Maybe we should have just deleted the thing in the first place. Will fix properly.